ramls/move.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Ids holder for moving items between holdings",
+  "type": "object",
+  "properties": {
+    "to": {
+      "type": "string",
```

Comment:
Does uuid.json already contain variable type? If yes we actually don't need it here.

Reply:
removed